schemas/EiffelMetaProperty/1.0.0.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$"
+    },
+    "type": {
+      "type": "string",
+      "pattern": "^Eiffel[a-zA-Z]*Event$"
+    },
+    "version": {
+      "type": "string",
+      "pattern": "^[1-9][0-9]*\\.[0-9]+\\.[0-9]+",
+      "default": "3.0.0"
```

Comment:
Should we really have a default value?

Reply:
This is copy/paste from the schema of the individual events. Do you want to change it in this PR?

Reply:
It would make sense when describing each event on its own but it feels strange that the default for all events should be 3.0

Reply:
I guess this will cause issues for libraries that want to generate events based on schema? They would all have version 3.0.0 forever. Unless we create one EiffelMetaProperty schema for each meta.version value, and reference that.